solidIndex/index.html

```diff
@@ -0,0 +1,1603 @@
+<!DOCTYPE html>
```

Comment:
I think there might be a better filename than `solidIndex`.

Shall we call this document `type-index` or `type-indexes`?

Reply:
At the last meeting we agreed to move this whole part in its own repo. A repo we can name - PROPOSAL: Solid Indexes - and type index will be the specific one described for now in the first version.

Reply:
Decision was done in [last meeting](https://github.com/solid/webid-profile/blob/main/meetings/2022-08-30.md#separation-of-index-content) to have the repo named type-indexes.